domain/src/main/kotlin/jp/co/yumemi/android/code_check/domain/repositories/SearchRepository.kt

```diff
@@ -1,6 +1,6 @@
 package jp.co.yumemi.android.code_check.domain.repositories
 
-import jp.co.yumemi.android.code_check.domain.core.Pageable
+import jp.co.yumemi.android.code_check.domain.entities.Pageable
```

Comment:
<table data-meta="generated_by_./Dangerfile">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/gmvalentino8/github-sample-project/blob/562109cc5a9348054f8c0891746241d13acaf72a/domain/src/main/kotlin/jp/co/yumemi/android/code_check/domain/repositories/SearchRepository.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>